types/artifacts.d.ts

```diff
@@ -29,6 +29,8 @@ declare global {
       LighthouseRunWarnings: string[];
       /** Whether the page was loaded on either a real or emulated mobile device. */
       TestedAsMobileDevice: boolean;
+      /** Device which Chrome is running on. */
+      HostFormFactor: 'desktop'|'mobile';
```

Comment:
`TestedAsMobileDevice`
`HostUserAgent`
`NetworkUserAgent`
`HostFormFactor`
`emulatedFormFactor`
`deviceScreenEmulationMethod`

https://www.theonion.com/taco-bells-five-ingredients-combined-in-totally-new-way-1819564909